app/javascript/searchableItemList/searchableItemList.js

```diff
@@ -130,7 +130,7 @@ export function loadNextPage() {
   component.setState({ page: page + 1 });
   component.search(query, {
     page: page + 1,
-    tags: [selectedTag],
+    tags: selectedTag ? [selectedTag] : [],
```

Comment:
**suggestion (non-blocking):** If we're only guarding against `selectedTag` being `nil` we could do that without the ternary:

```suggestion
    tags: [selectedTag].compact,
```

Reply:
I wouldn't find the `.compact` method in JS :)

Reply:
OMG, I was still a bit sleepy when I wrote this, this isn't actually Ruby lol